product-specifications/example-2.md

```diff
@@ -0,0 +1,21 @@
+# Interactive Animal Identification Tool (IAIT)
+
+This tool will help users identify animals by asking a series of questions and then provide a list of most likely animals based on the user's responses. This tool will be used by wildlife conservationists to identify animals they encounter.
+
+## User Flow
+
+1. Users answer a series of questions about the observed animal:
+    - What color is the animal?
+    - Where was the animal observed?
+    - How large was the animal?
```

Comment:
Could we ask how the animal primarily moves? 
Also what the animal primarily eats. (herbivore, omnivore or carnivore)
What kind of noise does the animal make?

Reply:
All great ideas!